### PR TITLE
update link for examples

### PR DESCRIPTION
**Pull request by matiasbenary**

### CODE REVIEW

**Review by gagdiez — CHANGES_REQUESTED** [review]
most of the links should be "context aware", for example:
- if we talk about the JS contract, then we need to point to the contract
- if we talk about the frontend, then we need to point to the frontend

File: docs/3.tutorials/crosswords/02-beginner/05-logging-in.md [hunk 1]
```diff
@@ -157,7 +157,7 @@ So signing a message is fine, but what if we could do better?
 
 With NEAR, we can leverage access keys to improve a user's login experience and give the power back to the user.
 
-If I log into the [Guest Book example site](https://github.com/near-examples/guest-book-js), I create a unique key just for that dApp, adding it to my account. When I'm done I can remove the key myself. If I suspect someone has control of my key (if a laptop is stolen, for example) I can remove the key as long as I have a full-access key in my control.
+If I log into the [Guest Book example site](https://github.com/near-examples/guest-book-examples), I create a unique key just for that dApp, adding it to my account. When I'm done I can remove the key myself. If I suspect someone has control of my key (if a laptop is stolen, for example) I can remove the key as long as I have a full-access key in my control.
```
Comment (matiasbenary) [inline docs/3.tutorials/crosswords/02-beginner/05-logging-in.md:160]:
```suggestion
If I log into the [Guest Book example site](https://github.com/near-examples/guest-book-examples/tree/main/frontend), I create a unique key just for that dApp, adding it to my account. When I'm done I can remove the key myself. If I suspect someone has control of my key (if a laptop is stolen, for example) I can remove the key as long as I have a full-access key in my control.
```

File: docs/sdk/js/intro.md [hunk 2]
```diff
@@ -35,7 +35,7 @@ The best way to create a new NEAR app connected with a frontend is through [crea
 npx create-near-app
 ```
 
-If you only wish to develop and deploy a JS contract, the [`hello-near-js`](https://github.com/near-examples/hello-near-js) repository is great to use as a template or one of the [examples in the SDK repository](https://github.com/near/near-sdk-js/tree/develop/examples/src).
+If you only wish to develop and deploy a JS contract, the [`hello-near-js`](https://github.com/near-examples/hello-near-examples) repository is great to use as a template or one of the [examples in the SDK repository](https://github.com/near/near-sdk-js/tree/develop/examples/src).
```
Comment (matiasbenary) [inline docs/sdk/js/intro.md:38]:
```suggestion
If you only wish to develop and deploy a JS contract, the [`hello-near-ts`](https://github.com/near-examples/hello-near-examples/tree/main/contract-ts) repository is great to use as a template or one of the [examples in the SDK repository](https://github.com/near/near-sdk-js/tree/develop/examples/src).
```

File: docs/3.tutorials/examples/donation.md [hunk 3]
```diff
@@ -2,144 +2,213 @@
...
 
 ---
 
-## Starting the Donation Example
+## Structure of the Example
+
+The example is divided in two main components:
 
-You have two options to start the Donation Example. The first and recommended is to use the app through Gitpod, which will open a web-based interactive environment. The second option is to clone the repository locally, for which you will need to install all the [Prerequisites](../../2.develop/prerequisites.md).
+1. The smart contract, available in two flavours: rust and javascript
```
Comment (gagdiez) [inline docs/3.tutorials/examples/donation.md:34]:
```suggestion
1. The smart contract, available in two flavors: rust and javascript
```

**Review by gagdiez — APPROVED** [review]

**Pull request merged**